openedx/features/partners/tests/test_login.py

```diff
@@ -0,0 +1,96 @@
+from django.urls import reverse
+from openedx.features.partners.tests.factories import PartnerUserFactory, PartnerFactory
+from rest_framework.test import APITestCase
+from rest_framework import status
+from student.tests.factories import UserFactory
```

Comment:
Let's rearrange imports into group as per pep8 and our style and then in alphabetical order

Reply:
Done.